development/README.md

```diff
@@ -282,4 +282,191 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 
 ## Configure/build/install QGIS in Qt Creator.app
 
-TODO
+### Install QtCreator 4.0 or better
+
+* Grab the installer from https://www.qt.io/download-open-source/
+* Run the installer. You ONLY need the qt-creator package so disable the Qt5 and don’t install Qt libs (we will use the qt4 installed by brew)
+* Once the installer is run open qt-creator
+* Make a directory in your QGIS source tree called ``build``
```

Comment:
Btw, you can change the default build directory in Prefs -> Build & Run -> General. I usually set it to `../%{CurrentProject:Name}_build` to keep it out out source, and it usually sorts right next to the project in Finder windows. Since I almost always do RelWithDebInfo builds on Mac, having a separate build directory, per build type, is not an concern.